### PR TITLE
Release/mip9.0.3

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:
- Rearrangement of chromograph recipes and changes to how the outfiles are presented in the deliverables file
- SpliceAI annotation with VEP instead of vcfanno
- Sample specific naming of rhocall viz outfiles
- Correct presentation of upd outfiles in the deliverables file

### How to test:

- Automatic and continuous test suite
- Verification with NIST sample

### Expected outcome:
- [ ] Installation, unit and integration test suite pass
- [ ] NIST sample passes verification

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Chromograph.pm [hunk 1]
```diff
@@ -166,23 +165,24 @@ sub analysis_chromograph {
         }
     );
 
+    my @contigs               = grep { !/M/xms } @{ $file_info_href->{contigs} };
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Chromograph.pm:168]: Could you use sub "delete_contig_elements". It is a bit misnamed cause it does not delete the original array element but the return has the element deleted

File: lib/MIP/Recipes/Analysis/Chromograph.pm [hunk 2]
```diff
@@ -426,26 +413,31 @@ sub analysis_chromograph_proband {
         }
     );
 
-    ## Switch from case id to sample id for the outfiles since this is done per sample
-    $infile_name_prefix =~ s/$case_id/$sample_id/xms;
+    my @outfile_name_prefixes;
+    my @contigs = grep { !/M/xms } @{ $file_info_href->{contigs} };
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Chromograph.pm:417]: Same here

File: lib/MIP/Recipes/Analysis/Mip_vcfparser.pm [hunk 3]
```diff
@@ -1047,12 +1047,13 @@ sub analysis_mip_vcfparser_sv_wes {
...
-            parse_vep            => $active_parameter_href->{sv_varianteffectpredictor},
-            per_gene             => $active_parameter_href->{sv_vcfparser_per_gene},
-            pli_values_file_path => $active_parameter_href->{vep_plugin}{ExACpLI}{path},
+            filehandle    => $filehandle,
+            infile_path   => $infile_path,
+            log_file_path => $log_file_path,
+            parse_vep     => $active_parameter_href->{sv_varianteffectpredictor},
+            per_gene      => $active_parameter_href->{sv_vcfparser_per_gene},
+            pli_values_file_path =>
+              $active_parameter_href->{vep_plugin}{ExACpLI}{parameters}[0],
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Mip_vcfparser.pm:1056]: Plugin controller?

File: lib/MIP/Recipes/Analysis/Mip_vcfparser.pm [hunk 4]
```diff
@@ -1411,7 +1412,7 @@ sub analysis_mip_vcfparser_sv_wgs {
                 parse_vep     => $active_parameter_href->{sv_varianteffectpredictor},
                 per_gene      => $active_parameter_href->{sv_vcfparser_per_gene},
                 pli_values_file_path =>
-                  $active_parameter_href->{vep_plugin}{ExACpLI}{path},
+                  $active_parameter_href->{vep_plugin}{ExACpLI}{parameters}[0],
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Mip_vcfparser.pm:1415]: Plugin controller?

File: templates/mip_download_rd_dna_config_-1.0-.yaml [hunk 5]
```diff
@@ -757,20 +757,20 @@ reference_feature:
         url_prefix: https://storage.googleapis.com/genomics-public-data/resources/broad/hg38/v0/
   rank_model:
     grch37:
```
Comment (henrikstranneheim) [inline templates/mip_download_rd_dna_config_-1.0-.yaml:759]: Download of "grch37_vcfanno_config_-v1.13-.toml"

File: templates/mip_dragen_rd_dna_config.yaml [hunk 6]
```diff
@@ -22,7 +22,7 @@ sample_info_file: cluster_constant_path!/case_id!/analysis_constant_path!/case_i
 ## References
 genmod_models_reduced_penetrance_file: grch37_cust003-cmms-red-pen_-2017-.tsv
 human_genome_reference: grch37_homo_sapiens_-d5-.fasta
-rank_model_file: rank_model_-v1.29-.ini
+rank_model_file: rank_model_-v1.30-.ini
```
Comment (henrikstranneheim) [inline templates/mip_dragen_rd_dna_config.yaml:25]:
```suggestion
rank_model_file: rank_model_-v1.31-.ini
```

File: templates/mip_rd_dna_config.yaml [hunk 7]
```diff
@@ -41,7 +41,7 @@ sv_svdb_query_db_files:
 qccollect_eval_metric_file: qc_eval_metric_-v1.2-.yaml
 qccollect_regexp_file: qc_regexp_-v1.26-.yaml
 vcf2cytosure_blacklist: grch37_cytosure_blacklist_-1.0-.bed
```
Comment (henrikstranneheim) [inline templates/mip_rd_dna_config.yaml:43]: What about the rank model update (30 -> 31) for this one?

File: templates/mip_rd_dna_panel_config.yaml [hunk 8]
```diff
@@ -24,9 +24,9 @@ genmod_models_reduced_penetrance_file: grch37_cust003-cmms-red-pen_-2017-.tsv
 human_genome_reference: grch37_homo_sapiens_-d5-.fasta
 qccollect_eval_metric_file: qc_eval_metric_-v1.2-.yaml
 qccollect_regexp_file: qc_regexp_-v1.26-.yaml
-rank_model_file: rank_model_-v1.29-.ini
+rank_model_file: rank_model_-v1.30-.ini
```
Comment (henrikstranneheim) [inline templates/mip_rd_dna_panel_config.yaml:27]:
```suggestion
rank_model_file: rank_model_-v1.31-.ini
```

File: templates/mip_rd_dna_vcf_rerun_config.yaml [hunk 9]
```diff
@@ -22,7 +22,7 @@ sample_info_file: cluster_constant_path!/case_id!/analysis_constant_path!/case_i
 ## References
 genmod_models_reduced_penetrance_file: grch37_cust003-cmms-red-pen_-2017-.tsv
 human_genome_reference: grch37_homo_sapiens_-d5-.fasta
-rank_model_file: rank_model_-v1.29-.ini
+rank_model_file: rank_model_-v1.30-.ini
```
Comment (henrikstranneheim) [inline templates/mip_rd_dna_vcf_rerun_config.yaml:25]:
```suggestion
rank_model_file: rank_model_-v1.31-.ini
```

**Review by henrikstranneheim — APPROVED** [review]
Excellent!

File: lib/MIP/Recipes/Analysis/Chromograph.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Chromograph.pm:168]: Good idea!

**Review by henrikstranneheim — DISMISSED** [review]
@jemten great work! A lot of nice improvements - well done!

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**